src/homework4/ex5/ContainGivenString.java

```diff
@@ -0,0 +1,22 @@
+package homework4.ex5;
+
+import java.util.Scanner;
+
+/**
+ * @author Gohar Farmanyan
+ */
+public class ContainGivenString {
+    public static void main(String args[]) {
+        Scanner scanner = new Scanner(System.in);
```

Comment:
hmm, esi function-ov aneir, vor ham karanair test anel, ham el null u empty-in el stugeir)